app/federation-v2/server/matrix-client/user.ts

```diff
@@ -33,40 +49,29 @@ export const invite = async (inviterId: string, roomId: string, invitedId: strin
 		console.log(`[${inviterId}-${invitedId}-${roomId}] Remote room created as ${matrixRoomId}...`);
 	}
 
-	// Find the inviter user
-	let inviterUser = MatrixBridgedUser.getById(inviterId);
-
-	if (!inviterUser) {
-		console.log(`[${inviterId}-${invitedId}-${roomId}] Creating remote inviter user...`);
-
-		// Get the user
-		const user = await dataInterface.user(inviterId);
-
-		// Create the missing user
-		inviterUser = await matrixClient.user.createRemote(user);
-
-		console.log(`[${inviterId}-${invitedId}-${roomId}] Inviter user created as ${inviterUser.mui}...`);
-	}
-
-	// Normalize the user situation
-	// If the user does not exist locally, we might need to create one
-
 	// Determine if the user is local or remote
 	let invitedUserMatrixId = invitedId;
 	const invitedUserDomain = invitedId.includes(':') ? invitedId.split(':').pop() : '';
 	const invitedUserIsRemote = invitedUserDomain && invitedUserDomain !== getConfig().homeserverDomain;
 
 	console.log(invitedUserMatrixId, invitedUserDomain, invitedUserIsRemote);
 
+	// Find the invited user in Rocket.Chats users
+	let invitedUser = Users.findOneByUsername(invitedId.replace('@', ''));
+
+	if (!invitedUser) {
+		// Create the invited user
+		invitedUser = await matrixClient.user.createLocal(invitedUserMatrixId);
+	}
```

Comment:
This was important to make sure regardless of where the user existed wether local matrix server or a remote one that they had a user in Rocket.Chat